Ex_23/index1.js

```diff
@@ -0,0 +1,12 @@
+'use striect'
+let curHour = Number(prompt ('Please, enter fwtat hour is it (from 0-23)', 100));
+if (curHour < 0 || curHour > 23) {
+    alert ("Your input incorrect");
+}
+else if (curHour < 8 || curHour == 18 ) {
+    alert ('It is not working hours');
+}
+else (curHour > 8 || curHour == 8 && curHour < 18) ;
+{
+    alert ('We are open!!!');
+}
```

Comment:
Why you have ex_22 and ex_23 looking same?
ex 22 should be "Comparing 2 numbers, using conditions"
ex 23 should be "Working hours, using conditions and logical operators".